packages/shared/package.json

```diff
@@ -41,7 +41,6 @@
     },
     "dependencies": {
         "@deriv/api-types": "^1.0.94",
-        "@deriv/stores": "^1.0.0",
```

Comment:
This has been added in #7307 but it's unused. On the other hand, it's creating a circular dependency hence removing it.

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/72082844/235844152-e0983309-6a9d-4007-b5fc-ebe0c5b78f46.png">